erc1155/src/lib.rs

```diff
@@ -280,116 +302,142 @@ pub unsafe extern "C" fn meta_state() -> *mut [i32; 2] {
 pub unsafe extern "C" fn handle() {
     let action: Action = msg::load().expect("Could not load Action");
     match action {
-        Action::Mint(account, id, amount) => {
-            ERC1155_TOKEN.mint(&account, &id, amount);
-            let transfer_data = TransferSingleReply {
-                operator: msg::source(),
-                from: ZERO_ID,
-                to: account,
-                id,
-                amount,
-            };
-
-            msg::reply(Event::TransferSingle(transfer_data), 0);
+        Action::Mint(account, id, amount, meta) => {
+            ERC1155_TOKEN.mint(&account, &id, amount, meta);
+            msg::reply(
```

Comment:
I think it's better to write the message reply to the the implementation of functions